src/applications/personalization/dashboard/components/claims-and-appeals/ClaimsAndAppeals.jsx

```diff
@@ -158,8 +163,8 @@ const hasAppealsErrorSelector = state => {
 const mapStateToProps = state => {
   const claimsState = state.disability.status;
   const claimsV2Root = claimsState.claimsV2;
-  const appealsLoading = claimsV2Root.appealsLoading;
-  const claimsLoading = claimsV2Root.claimsLoading;
+  const { appealsLoading } = claimsV2Root;
```

Comment:
I’m sure this change is just to appease the linter, but it should probably be
```
const { appealsLoading, claimsLoading } = claimsV2Root;
```

Reply:
This was an auto modification from my webstorm. Sorry I will get it fixed.

Reply:
No worries, I figured it was auto-fixed 👍